docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,26 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting server icon
+
+To get server icon URI from the client it has to be set by the server beforehand. Trying to read
```

Comment:
Can we rephrase as:
To retrieve the server icon URI from the client, the server must set the URI in advance. If you try to read the server icon URI before it is set, the result is a null value.

Please verify whether the technical meaning is altered.

Reply:
done

Reply:
In line no: 161, this comment is not updated.

Reply:
Right now this paragraph has the following form:

"Please follow the example presented below to retrieve the server icon URI on the client side.                                                           
If you try to retrieve the server icon URI before it was set, the result will be a null value. "

Please let me know what information is missing from it, because i think its equivalent to what you proposed.